apps/server/src/modules/learnroom/controller/api-test/dashboard.api.spec.ts

```diff
@@ -71,7 +71,7 @@ describe('Dashboard Controller (API)', () => {
 			const { id: dashboardId } = await dashboardRepo.getUsersDashboard(user.id);
 			currentUser = mapUserToCurrentUser(user);
 
-			const response = await request(app.getHttpServer()).get('/dashboard');
+			const response = await request(app.getHttpServer()).get('/dashboard?showSubstitute=false');
```

Comment:
what about a case with showSubtitue true, and also consider a case for student role?

Reply:
done